# ArMarkers.msg

uint16 		markerNo
int32[] 	markersIds
float64[] 	rVecs		# contains all rvec in sequence (3*markerNo items)
float64[] 	tVecs		# contains all tvec in sequence (3*markerNo items)

# !
# matrixes are not supported
#